packages/botonic-nlp/tests/tasks/ner/process/processor.test.ts

```diff
@@ -1,31 +1,83 @@
 import { PADDING_TOKEN } from '../../../../src/preprocess/constants'
 import { Processor } from '../../../../src/tasks/ner/process/processor'
-import * as helper from '../../../helpers/tools-helper'
+import * as constantsHelper from '../../../helpers/constants-helper'
+import * as toolsHelper from '../../../helpers/tools-helper'
 
 describe('NER Processor', () => {
   const sut = new Processor(
-    helper.preprocessor,
-    helper.tokenEncoder,
-    helper.entitiesEncoder
+    toolsHelper.preprocessor,
+    toolsHelper.tokenEncoder,
+    toolsHelper.entitiesEncoder
   )
-  test('Process sample', () => {
-    expect(
-      sut
-        .process([
-          { class: '', entities: [], text: 'I love this leather jacket' },
-        ])
-        .x.arraySync()
-    ).toEqual([[6, 11, 1, 27, 0, 0, 0, 0, 0, 0, 0, 0]])
+  // The samples text has been chosen so we have a short and a long sentence.
```

Comment:
always use code if possible instead of comments.
eg.
shortTextSample = {class: ,...}
longTextSample = {class: ,...}
sut.process([shortTextSample, longTextSample])

in the comments you should always write WHY (if necessary).
in this why is it important to test both short & long text? or did you mean single vs multiple sentences?